packages/graphql-validate-fixtures/README.md

```diff
@@ -23,13 +23,76 @@ In order to associate a fixture with a GraphQL query or mutation in your app, yo
 
 Once this is done, you can validate your fixtures using the CLI or Node.js API.
 
+### Configuration
+
+This tool reads schema information from a [`.graphqlconfig`](https://github.com/prisma/graphql-config) file in the project root. The configuration can contain one nameless project or many named projects. The configuration is compatible with the [vscode-graphql extension](https://github.com/prisma/vscode-graphql). This extension provides syntax highlighting and autocomplete suggestions for graphql files.
+
+Each project specifies a `schemaPath`, `include`, and `exclude` globs. Glob patterns match paths relative to the location of the configuration file. Omit `exclude` if empty.
+
+On startup this tool performs the following actions:
+
+* Loads all schemas
+* Discovers all operations belonging to each schema
+* Discovers all fixtures and infers operation names as described [above](#Usage)
+* Validates fixtures against the operation with a matching name
+  * Reports operation not found error if no schema matches
+  * Reports ambiguous operation name error if more than one schema matches
+
+See the [official specification documentation](https://github.com/prisma/graphql-config/blob/master/specification.md#use-cases) for more detail and examples.
+
+#### Examples
+
+A single nameless project configuration
+
+```json
+{
+  "schemaPath": "build/schema.json",
+  "includes": "app/**/*.graphql"
+}
+```
+
+A multi-project configuration
+
+```json
+{
+  "projects": {
+    "foo": {
+      "schemaPath": "build/schema/foo.json",
+      "includes": "app/foo/**/*.graphql"
+    },
+    "bar": {
+      "schemaPath": "build/schema/bar.json",
+      "includes": "app/bar/**/*.graphql"
+    }
+  }
+}
+```
+
+A project configuration with a `schemaTypesPath` override
+
+```json
+{
+  "projects": {
+    "foo": {
+      "schemaPath": "build/schema/foo.json",
+      "includes": "app/foo/**/*.graphql"
+    },
+    "bar": {
+      "schemaPath": "build/schema/bar.json",
+      "includes": "app/bar/**/*.graphql",
+      "extensions": {
+        "schemaTypesPath": "app/bar/types/graphql.ts"
+      }
+    }
+  }
+}
+```
```

Comment:
Do we need all this stuff in this README? Seems like it should be in one place (probably `GraphQL-tool-utilities` and every consuming package just makes a passing mention and links to the full details?

Reply:
The utilities package is published too, IMO it is not a problem at all to just link to its docs on configuration for the project.

Reply:
ok, i'll refactor this into the utilities package then.

Reply:
https://github.com/Shopify/graphql-tools-web/pull/29 covers the main refactoring of the documentation